pkg/operator/csidriveroperator/crcontroller.go

```diff
@@ -200,25 +178,7 @@ func (c *CSIDriverOperatorCRController) applyClusterCSIDriver(requiredOriginal *
 		return nil, false, err
 	}
 
-	modified := resourcemerge.BoolPtr(false)
-	existingCopy := existing.DeepCopy()
-
-	resourcemerge.EnsureObjectMeta(modified, &existingCopy.ObjectMeta, required.ObjectMeta)
```

Comment:
I think we might want to merge metadata of an object in-case there are some cluster-wide annotations applied to these CRs, but if we come to that we can also push and update and do metadata merging later on (so I am bit torn).